tests/hikari/impl/test_entity_factory.py

```diff
@@ -4314,7 +4321,7 @@ def test_deserialize_message_with_other_unset_fields(self, entity_factory_impl,
         assert message.application.cover_image_hash is None
         assert message.application.icon_hash is None
         assert message.referenced_message is None
-        assert message.member is undefined.UNDEFINED
+        assert message.member is None
```

Comment:
`referenced_message` should also be fixed, it should never be undefined for a full Message

Reply:
Looks like attrs handles re-assigning the attributes in a really nice way (mostly not editing them if it doesnt have to) and it makes mypy happy, so I think we will stick with it